pkgs/tools/misc/rtx/default.nix

```
@@ -0,0 +1,72 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, installShellFiles
+, stdenvNoCC
+, perl
+, coreutils
+, bash
+, direnv
+, Security
+}:
+
+let
+  owner = "jdxcode";
+  pname = "rtx";
+  version = "1.29.7";
+in
+rustPlatform.buildRustPackage rec {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-w61ik1Tne7cO1Aez4aem3/UQfWjoCTnz+vR1YaaHDhY=";
+  };
+
+  cargoSha256 = "sha256-PGq/LSM8bsNWRVDLLhaiEgaXIN+957HnF0SUx6M7Wus=";
+
+  # perl because: https://docs.rs/openssl/0.10.45/openssl/#vendored
+  nativeBuildInputs = [ perl installShellFiles ];
+  buildInputs = lib.optionals stdenvNoCC.isDarwin [ Security ];
```

```suggestion
  buildInputs = lib.optionals stdenv.isDarwin [ Security ];
```
stdenv is fine to use since we only evaluate it and don't build anything from it